test/common/router/config_impl_test.cc

```diff
@@ -2362,6 +2362,92 @@ TEST(RouteMatcherTest, Retry) {
                 .retryOn());
 }
 
+TEST(RouteMatcherTest, RetryVirtualHostLevel) {
+  const std::string json = R"EOF(
+{
+  "virtual_hosts": [
+    {
+      "name": "www2",
+      "domains": ["www.lyft.com"],
+      "retry_policy": {
+        "per_try_timeout_ms" : 1000,
```

Comment:
v2 YAML for all tests going forward, thanks! :)

Reply:
ah yes, i'll change that